diff -up anbox-master/external/cpu_features/CMakeLists.txt.3~ anbox-master/external/cpu_features/CMakeLists.txt
--- anbox-master/external/cpu_features/CMakeLists.txt.3~	2020-09-05 05:39:11.601037665 +0200
+++ anbox-master/external/cpu_features/CMakeLists.txt	2020-09-05 05:39:44.944762024 +0200
@@ -128,7 +128,7 @@ if(NOT PROCESSOR_IS_X86 AND UNIX)
   list(APPEND CPU_FEATURES_SRCS $<TARGET_OBJECTS:unix_based_hardware_detection>)
 endif()
 add_library(cpu_features ${CPU_FEATURES_HDRS} ${CPU_FEATURES_SRCS})
-set_target_properties(cpu_features PROPERTIES PUBLIC_HEADER "${CPU_FEATURES_HDRS}")
+set_target_properties(cpu_features PROPERTIES PUBLIC_HEADER "${CPU_FEATURES_HDRS}" SOVERSION 0 VERSION "0.0")
 setup_include_and_definitions(cpu_features)
 target_link_libraries(cpu_features PUBLIC ${CMAKE_DL_LIBS})
 set_property(TARGET cpu_features PROPERTY POSITION_INDEPENDENT_CODE ${BUILD_PIC})
